### PR TITLE
Added internal setting to enable/disable headers storage

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #...
| License       | MIT

Related to #3108. 
I added a setting to enable/disable the http headers storage (and I also added missing translations).

### CODE REVIEW

**Comment by tcitworld** [comment]
Maybe add some documentation ? Why activate this setting, what will be saved, ...

**Comment by nicosomb** [comment]
About the documentation, you're right. I'll add it into the correct repo.

File: src/Wallabag/CoreBundle/Helper/ContentProxy.php [hunk 1]
```diff
@@ -24,15 +24,17 @@ class ContentProxy
     protected $mimeGuesser;
     protected $fetchingErrorMessage;
     protected $eventDispatcher;
+    protected $storeArticleHeaders;
 
-    public function __construct(Graby $graby, RuleBasedTagger $tagger, ValidatorInterface $validator, LoggerInterface $logger, $fetchingErrorMessage)
+    public function __construct(Graby $graby, RuleBasedTagger $tagger, ValidatorInterface $validator, LoggerInterface $logger, $fetchingErrorMessage, $storeArticleHeaders)
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Helper/ContentProxy.php]: I'll set it to false by default to avoid editing all tests files

**Comment by nicosomb** [comment]
Thank you @j0k3r.

**Pull request merged**